earthkit/geo/distance.py

```diff
@@ -187,31 +208,33 @@ def __init__(self, lats, lons):
             lats = lats[mask]
             lons = lons[mask]
 
-        x, y, z = ll_to_xyz(lats, lons)
+        x, y, z = _ll_to_xyz(lats, lons, radius=1)
         v = np.column_stack((x, y, z))
```

Comment:
ll_to_xyz maybe renamed to latlon_to_xyz; in eckit::geo the API order is PointLonLat because it maps straight to x,y, but here in a user-facing library the typical order is lat,lon

Reply:
Done.